packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -37,18 +41,20 @@ const CupertinoDynamicColor _kThumbColor = CupertinoDynamicColor.withBrightness(
   darkColor: Color(0xFF636366),
 );
 
+// The height of the separator.
+const double _kSeparatorHeight = 18.0;
```

Comment:
nit: If both the height and the width are const, turn them into a `Size` const?

Reply:
There are several places that width and height are used separately, maybe just keep them as is?

Reply:
I didn't realize those are only used directly by the widgets / render objects. They are used to construct other constants it looks like?

Reply:
I'm a bit confused by `_kSeparatorHeight`. It looks like it's not used directly but used to compute `_kSeparatorInset`. So the real semantics of `_kSeparatorHeight` is "the height of the separator when the segmented control is at the min height"?

Reply:
Ah you are right. I was thinking providing a const height value might be more straightforward to understand. But it seems only "inset" doesn't change, when segments height have some larger values, the separator height is also getting larger. Updated.